docs/metal_interconnections_create.md

```diff
@@ -24,14 +24,17 @@ metal interconnections create -n <name> [-m <metro>] [-r <redundancy> ] [-t <typ
 ### Options
 
 ```
-  -h, --help                    help for create
-  -m, --metro string            metro in the interconnection
-  -n, --name string             Name of the interconnection
-  -O, --organizationID string   Org ID
-  -p, --projectID string        project ID
-  -r, --redundancy string       Website URL of the organization.
-  -t, --type string             type of of interconnection.
-  -v, --vrfs strings            Return only the specified vrfs.
+  -h, --help                        help for create
+  -m, --metro string                metro in the interconnection
+  -n, --name string                 Name of the interconnection
+  -O, --organizationID string       Org ID
+  -p, --projectID string            project ID
+  -r, --redundancy string           Website URL of the organization.
+  -T, --service-token-type string   Type of service token for shared connection. Enum: 'a_side', 'z_side'
+      --speed int32                 the maximum speed of the interconnections (default 1000000000)
+  -t, --type string                 type of of interconnection.
+      --vlans int32Slice            Array of int vLANs (default [])
```

Comment:
We generally use `VLANs` (I don't know what's right or best)

Reply:
`Array of int VLANs` -> `A list of VLANs to attach to the Interconnection`

We'll want to include an example snippet showing how to define this.

Reply:
Done